Classes/Domain/Repository/ExtensionRepository.php

```diff
@@ -61,20 +67,23 @@ public function findAll()
 
     /**
      * @param \EBT\ExtensionBuilder\Domain\Model\Extension $extension
+     *
+     * @throws \Exception
```

Comment:
this looks strange, maybe there is a nested error somewhere because throwing a top-level `\Exception` is the very worst choice in general.

Reply:
This comes from ExtensionBuilderConfigurationManager::getConfigurationFromModeler

Reply:
I would really urge to change this to something useful. at least a `RuntimeException`

Reply:
There are many places where a `\Exception` is thrown. What should I do with them?

Reply:
okay, didn't look into the code yet, if there are many, I suggest then leaving things as they are now and doing improvements in a an extra patch

Reply:
ok, then this pr is ready to be merged.